preload.js

```diff
@@ -362,6 +362,14 @@ setInterval(() => {
   window.nodeSetImmediate(() => {});
 }, 1000);
 
+// Set server-client time difference
+(async function setClockParams() {
+  window.maxTimeDifferential = 30;
```

Comment:
This variable doesn't have to be global, it is only used below.

Reply:
What if someone wants to check these vals somewhere else in code?
I'll make them local for now